.babelrc.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   ignore: [
     './lib/codemod/src/transforms/__testfixtures__',
     './lib/postinstall/src/__testfixtures__',
+    '**/typings.d.ts',
```

Comment:
This is a typings file, it has no runtime elements, and so babel doesn't need to work on this file.